src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.test.tsx

```diff
@@ -177,3 +178,22 @@ it('delete everything', async () => {
     []
   )
 })
+
+it('isBoundingBoxTooSmall', () => {
+  const geometryTooSmall = {
+    x: 0,
+    y: 0,
+    height: 0,
+    width: 0,
+    type: 'RECTANGLE',
+  }
+  const geometryValid = {
+    x: 0,
+    y: 0,
+    height: 0.35,
+    width: 0.35,
+    type: 'RECTANGLE',
+  }
```

Comment:
You can use [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) when there are multiple data for the same test.